posthog/queries/base.py

```diff
@@ -40,21 +49,20 @@ def determine_compared_filter(filter) -> Filter:
 
 
 def convert_to_comparison(trend_entity: List[Dict[str, Any]], filter, label: str) -> List[Dict[str, Any]]:
-    for entity in trend_entity:
```

Comment:
Is this a pure refactor or did anything in here actually change besides getting rid of the side effect?

Reply:
The change here is not mutating the entities. I made the change due to test failure, but I’ll check to see if this actually was the case